.github/workflows/check-motoko-changes.yml

```
@@ -54,3 +58,11 @@ jobs:
           else
             echo "has_changelog_changes=true" >> $GITHUB_OUTPUT
           fi
+
+          # Check for mops dependency changes
+          if ! git diff --name-only $BASE_SHA $HEAD_SHA | grep -q '^mops\.toml$'; then
```

Why not tell `git` to focus on that file?
```suggestion
          if ! git diff --name-only $BASE_SHA $HEAD_SHA -- mops.toml
```
```suggestion
          if ! git diff --name-only $BASE_SHA $HEAD_SHA | grep -q '^mops\.toml$'; then
```

---

Good idea